core/common/assets/js/api/core/commands.js

```diff
@@ -257,33 +274,69 @@ export default class Commands extends elementorModules.Module {
 	 * @returns {boolean|*} results
 	 */
 	run( command, args = {} ) {
-		if ( ! this.beforeRun( command, args ) ) {
+		if ( ! this.validateRun( command, args ) ) {
 			return false;
 		}
 
-		const component = this.getComponent( command ),
-			container = component.getRootContainer();
+		this.beforeRun( command, args );
 
-		this.current[ container ] = command;
-		this.currentArgs[ container ] = args;
+		/**
+		 * @type {(CommandBase)|*}
+		 */
+		const instance = this.commands[ command ].apply( this.getComponent( command ), [ args ] );
```

Comment:
`apply` is not run the command?

Reply:
Its apply of java-script its create instance
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply